utilities/GeppettoConfiguration.json

```diff
@@ -7,20 +7,9 @@
     "rootRedirect":"",
     "noTest": false,
     "extensions": {
-        "geppetto-default/ComponentsInitialization": false,
-        "geppetto-osb/ComponentsInitialization": false,
-        "geppetto-vfb/ComponentsInitialization": false,
-        "geppetto-neuron/ComponentsInitialization": false,
-        "geppetto-netpyne/ComponentsInitialization": false,
-        "geppetto-hm/ComponentsInitialization": false,
         "geppetto-nwbexplorer/ComponentsInitialization": true
```

Comment:
@filippometacell should the extensions still be here post repos refactoring?

Reply:
No, neither should be there that file. Let's get rid of it.

Reply:
Now it's coming directly from the application root, which is specific by design. That file was intended to be copied into the webapp root from the setup script.